CHANGELOG-2.0.md

```diff
@@ -1,30 +1,16 @@
-# UPGRADE FROM `v1.13.3` TO `v2.0.0`
+# CHANGELOG FOR `2.0.X`
```

Comment:
I don't think it's worth to maintain a changelog for 2.x...

Reply:
Ok